starlette/exceptions.py

```diff
@@ -22,7 +22,12 @@ def __repr__(self) -> str:
 
 class ExceptionMiddleware:
     def __init__(
-        self, app: ASGIApp, handlers: dict = None, debug: bool = False
+        self,
+        app: ASGIApp,
+        handlers: typing.Mapping[
+            typing.Any, typing.Callable[[Request, Exception], Response]
+        ] = None,
```

Comment:
It's quite a complicated type, should we move it to `starlette.types`? Like `ExceptionHandler` type

Reply:
The current types on `types.py` are going to be removed at some point: https://github.com/encode/starlette/issues/1217

That file will only contain this type at some point. Is that a problem? :thinking:

Reply:
Didn't notice the Issue,  
This mapping type just seems a bit complicated to me to be copied multiple times

Reply:
One drawback I noticed in VScode, haven't really checked and there might be some workarounds, is that they don't show function signatures when using the aliases as you mention. Not sure if it happens on PyCharm and other tools.